src/api/api.ts

```diff
@@ -221,20 +223,35 @@ export class OpenSeaAPI {
     return response;
   }
 
+  /**
+   * Post collection offer
```

Comment:
This is not the right documentation....

Also can you make this follow the typedoc standard with @param @return, etc